HW1/P2/P2b.py

```diff
@@ -0,0 +1,16 @@
+from P2 import *
+
+# Your code here
+sc=SparkContext()
+xrand=range(2000)
+yrand=range(2000)
+random.shuffle(xrand)
+random.shuffle(yrand)
+x = sc.parallelize(xrand,10)
+y = sc.parallelize(yrand,10)
+xy = x.cartesian(y)
+result = xy.map(lambda t:[t,mandelbrot((t[0]/500.0)-2,(t[1]/500.0)-2)])
+draw_image(result)
```

Comment:
I would definitely recommend you add more comments to your code. Granted, in this case, your code was pretty simple, it may have been harder to understand long-term.